www/app/page.tsx

```diff
@@ -24,17 +23,23 @@ export default async function Home() {
   // Get initial subscription state
   let isSubscribed = false;
   let freeMessages = 0;
+  let chatAccess;
 
   if (process.env.NEXT_PUBLIC_STRIPE_ENABLED === 'false') {
+    // In development, get the real message count but override subscription status
+    const realChatAccess = await getChatAccess(supabase, user.id);
+    chatAccess = {
+      isSubscribed: true,
+      freeMessages: realChatAccess.freeMessages,
+      canChat: realChatAccess.isSubscribed || realChatAccess.freeMessages > 0 // Always allow chat in development
```

Comment:
This comment is in-accurate and the value can still be false. This not always going to default to true